src/components/button.js

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = props => {
-  const { name, color, wide } = props;
-  const classes = color === 'white' && wide ? 'white wide' : color;
-  return (
-    <div className={classes}>{name}</div>
-  );
-};
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.namePass = this.namePass.bind(this);
```

Comment:
`this.namePass` is throwing a `Cannot read property 'bind' of undefined`.
And this is happening because you commented the function`namePass(e)` below from line 10-13